### PR TITLE
add methods to save ApiContext to JSON and restore it from JSON [#9]

### LINKED ISSUE #9: Allow saving context to JSON and restoring from it
## How to Reproduce
1. Look at the methods of `ApiContext`

## What Should Happen
1. `toJson` and `fromJson` methods allowing to save context to or load from JSON are present

## What Happens
1. There are no `toJson` nor `fromJson`

### PR DESCRIPTION
**Pull request by dnl-blkv**
Fixes #9

### CODE REVIEW

File: src/main/java/com/bunq/sdk/context/ApiContext.java [hunk 1]
```diff
@@ -124,14 +123,20 @@ public static ApiContext restore(String fileName) {
...
+  /**
+   * Restores a context from a given JSON string.
+   */
+  public static ApiContext fromJson(String json) {
+    return gson.fromJson(json, ApiContext.class);
+  }
+
   private void initialize(String deviceDescription, List<String> permittedIps) {
     /* The calls below are order-sensitive: to initialize a Device Registration, we need an
      * Installation, and to initialize a Session we need a Device Registration. */
```
Comment (OGKevin) [inline src/main/java/com/bunq/sdk/context/ApiContext.java]: Whats the reason for this comment being under need the method and not above it ? 

Also shouldn't it be:
```
/**
*
*/
```

Comment (OGKevin) [inline src/main/java/com/bunq/sdk/context/ApiContext.java]: Ah `The calls below`... 🤔 explains why its below the method 😁 

But then the formatting ? Shouldn't it be as 👆

Comment (dnl-blkv) [inline src/main/java/com/bunq/sdk/context/ApiContext.java]: @OGKevin The format you've specified is Doc Block comment which can be extracted to a generated documentation. This comment is opposite. It is just a normal multi-line comment which is only needed to explain the internals of the method, not the method itself.

So, Java has:
```
/**
 * Doc Block
 */
```

and

```
/*
 * Just a multi-line comment.
 */

Comment (OGKevin) [inline src/main/java/com/bunq/sdk/context/ApiContext.java]: ack

**Pull request merged**